src/get-started/flutter-for/ios-devs.md

```diff
@@ -5,467 +5,836 @@ description: Learn how to apply iOS developer knowledge when building Flutter ap
 
-`StatelessWidgets` are useful when the part of the user interface you are
-describing does not depend on anything other than the initial configuration
-information in the widget.
+In both SwiftUI and Flutter, 
+layouts are composed by nesting views or widgets, 
+respectively, within one another.
 
-For example, in iOS, this is similar to placing a `UIImageView`
-with your logo as the `image`. If the logo is not changing during runtime,
-use a `StatelessWidget` in Flutter.
+### Layout process
 
-If you want to dynamically change the UI based on data received
-after making an HTTP call, use a `StatefulWidget`.
-After the HTTP call has completed, tell the Flutter framework
-that the widget’s `State` is updated, so it can update the UI.
+In SwiftUI, the layout process is composed of a few different steps:
 
-The important difference between stateless and
-stateful widgets is that `StatefulWidget`s have a `State` object
-that stores state data and carries it over across tree rebuilds,
-so it's not lost.
+1. The parent view proposes some size to the child view.
+1. The child decides the size it requires by going through 
+this same process with its own list of children. 
+It passes down the proposed size and 
+asks its child what size it wishes to be. 
+It then returns the size it requires to its parent.
+1. The parent lays out the child, respecting the size that the child requested.
 
-If you are in doubt, remember this rule:
-if a widget changes outside of the `build` method
-(because of runtime user interactions, for example),
-it’s stateful.
-If the widget never changes, once built, it's stateless.
-However, even if a widget is stateful, the containing parent widget
-can still be stateless if it isn’t itself reacting to those changes
-(or other inputs).
+Flutter follows a similar process, with a few notable differences:
+
+1. The parent widget passes constraints down to its children 
+(maximum height and width, and minimum height and width). 
+1. The child attempts to decide its size 
+by going through this same process with its own list of children. 
+It tells its child what its constraints are, 
+and asks it what size it wishes to be. 
+1. The parent lays out the child, ensuring that it’s within the original constraints.
+
+The main differences are that in Flutter the size can be asserted by 
+the parent widget, and overrule the child’s desired size. 
+Parents can force a child to be a specific size by using tight 
+constraints where the minimum height is equal to the maximum height, 
+or the minimum width is equal to the maximum width.
+
+In SwiftUI, views can choose to expand to the space that the parent 
+has said is available, or choose to be the size of its content. 
+Flutter widgets behave similarly. 
+However, parents can offer unbounded constraints, where one or both 
+of the maximum values is infinity. 
+In these cases, if the child is expanding, 
+you will get an overflow warning, as shown below:
```

Comment:
The next line is Dart code. I don't see the overflow warning.

Reply:
I was referring to the screenshot below it

Reply:
I am referring to the screenshot below the code (showing the output of the code)